lib/src/main/java/io/cloudquery/types/InetType.java

```diff
@@ -0,0 +1,42 @@
+package io.cloudquery.types;
+
+import org.apache.arrow.memory.BufferAllocator;
+import org.apache.arrow.vector.FieldVector;
+import org.apache.arrow.vector.types.pojo.ArrowType;
+import org.apache.arrow.vector.types.pojo.FieldType;
+
+public class InetType extends ArrowType.ExtensionType {
+    public static final InetType INSTANCE = new InetType();
+
+    @Override
+    public ArrowType storageType() {
+        return Binary.INSTANCE;
+    }
+
+    @Override
+    public String extensionName() {
+        return "inet";
+    }
+
+    @Override
+    public boolean extensionEquals(ExtensionType other) {
+        if (!(other instanceof InetType))
```

Comment:
wouldn't this work?
```suggestion
        return other instanceof InetType;
```

Reply:
Yes I think you are right. I was keeping with the style of other ArrowTypes, but I think they are generated code from [this block](https://github.com/apache/arrow/blob/main/java/vector/src/main/codegen/templates/ArrowType.java#L261-L272).

I'll at the change as part of the next PR.